std_msgs/Header header
mask_rcnn/Mask[] masks

